src/state/queries/profile.ts

```diff
@@ -426,3 +434,15 @@ export function* findAllProfilesInQueryData(
     }
   }
 }
+
+export function resetProfilePostsQueries(did: string, timeout = 0) {
```

Comment:
Can we move this into `post-feed`? We try to colocate all functions that affect a query in the file that controls the query